packages/calcite-components/src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -92,7 +95,14 @@ export class DropdownGroup {
       this.groupPosition > 0 ? <div class="dropdown-separator" role="separator" /> : null;
 
     return (
-      <Host aria-label={this.groupTitle} role="group">
+      <Host
+        {...{
+          "aria-label": this.groupTitle,
+          role: "menu",
+          id: "demo-menu-list",
```

Comment:
This id should already be set on connectedCallback correct? I don't think it needs to be set here. Its also not unique in this instance.

Reply:
Thanks for the catch!